apps/backend/mail/package.json

```diff
@@ -7,21 +7,26 @@
 		"email": "email dev --dir src/emails"
 	},
 	"dependencies": {
-		"@elysiajs/bearer": "^1.3.0",
-		"@elysiajs/cron": "^1.3.0",
-		"@elysiajs/server-timing": "^1.3.0",
-		"@elysiajs/swagger": "^1.3.1",
-		"@paralleldrive/cuid2": "^2.2.2",
-		"drizzle-orm": "^0.44.4",
-		"drizzle-typebox": "^0.3.3",
-		"elysia": "catalog:backend"
+		"@elysiajs/bearer": "catalog:backend",
+		"@elysiajs/cron": "catalog:backend",
+		"@elysiajs/server-timing": "catalog:backend",
+		"@elysiajs/swagger": "catalog:backend",
+		"@paralleldrive/cuid2": "catalog:backend",
+		"drizzle-orm": "catalog:db",
+		"drizzle-typebox": "catalog:db",
+		"elysia": "catalog:backend",
+		"@opentelemetry/exporter-trace-otlp-http": "catalog:backend",
+		"@opentelemetry/instrumentation-express": "catalog:backend",
+		"@opentelemetry/instrumentation-http": "catalog:backend",
+		"@opentelemetry/resources": "catalog:backend",
+		"@opentelemetry/sdk-trace-node": "catalog:backend",
+		"@opentelemetry/semantic-conventions": "catalog:backend"
 	},
 	"devDependencies": {
-		"@react-email/preview-server": "4.2.8",
-		"bun-types": "latest"
+		"@react-email/preview-server": "catalog:backend"
```

Comment:
add 	"bun-types": "latest" in the root package and add in the devDependencies above